content/index.ja.md

```diff
@@ -0,0 +1,162 @@
+---
+# Translation instructions are after the "#" character in this first section. They are comments that do not show up in the web page. You do not need to translate the instructions after "#".
+# In this first section, do not translate the words before a colon. For example, do not translate "title:". Do translate the text after "title:".
+
+title: "WCAG 2 概要"
+nav_title: "Web Content – WCAG 2"
+lang: ja  # Change "en" to the translated-language shortcode
+last_updated: 2024-04-04  # Put the date of this translation YYYY-MM-DD (with month in the middle)
+
+# translators: # remove from the beginning of this line and the lines below: "# " (the hash sign and the space)
+# - name: "Makoto Ueki" # Add one -name: line for every translator
+# - name: "Makoto Ueki"   # Replace Jan Doe with translator name
+# contributors:
+
+
+github:
+  repository: w3c/wai-intro-wcag
+  path: 'content/index.ja.md'  # Add the language shortcode to the middle of the filename, for example: content/index.fr.md
+
+permalink: /standards-guidelines/wcag/ja  # Add the language shortcode to the end, with no slash at end, for example: /link/to/page/fr
+ref: /standards-guidelines/wcag/  # Do not change this
+
+description: WCAG2.0、WCAG2.1、WCAG2.2を含む、国際標準のウェブコンテンツアクセシビリティガイドライン（WCAG）を紹介。WCAG文書は、ウェブコンテンツを障害がある人にとってよりアクセシブルにする方法を説明しています。
+image: /content-images/wai-intro-wcag/wcag-intro-social.jpg
+feedbackmail: wai@w3.org
+
+# In the footer below:
+# Do not change the dates
+# Translate the other words below, including "Date:" and "Editor:"
+# Translate the Working Group name. Leave the Working Group acronym in English.
+footer: >
+  <p><strong>最終更新日：</strong> 7 March 2024、初版公開 July 2005</p>
+  <p><strong>編集者：</strong> <a href="https://www.w3.org/People/Shawn/">Shawn Lawton Henry</a></p>
+  <p>教育・啓発ワーキンググループ (<a href="https://www.w3.org/WAI/about/groups/eowg/">EOWG</a>)及びアクセシビリティガイドライン ワーキンググループ (<a href="https://www.w3.org/WAI/about/groups/agwg/">AG WG</a>)の協力を得て作成されました。</p>
+---
+
+{::nomarkdown}
+{% include box.html type="start" h="2" title="Summary" class="full" %}
+{:/}
+
+このページでは、WCAG 2.0、WCAG 2.1、およびWCAG 2.2を含む、国際標準であるWebコンテンツ・アクセシビリティ・ガイドライン（WCAG）について紹介します。WCAG文書は、Webコンテンツを障害がある人にとってよりアクセシブルにする方法を説明しています。
+
+関連ページ [introduces WCAG 3＜英語＞](/standards-guidelines/wcag/wcag3-intro/).
+
+WCAGはアクセシビリティの入門書ではありません。入門書のようなドキュメントとしては、[Accessibility Fundamentals Overview＜英語＞](/fundamentals/) を参照してください。
```

Comment:
@MakotoUeki Please translate the text in the links and do not manually add `(in English)`, even for external links.

See context in https://www.w3.org/WAI/about/translating/guides/translation-guidance/#links

Reply:
@MakotoUeki Please note this applies to all links in the translation.